tests/test_misc.py

```diff
@@ -2156,8 +2141,6 @@ def test_not_announceable_spend(revault_network, bitcoind, executor):
         man.rpc.updatespendtx(spend_tx)
     spend_psbt = serializations.PSBT()
     spend_psbt.deserialize(spend_tx)
-    spend_psbt.tx.calc_sha256()
-    man.rpc.setspendtx(spend_psbt.tx.hash)
```

Comment:
If you do that the "One less output is ok though" part is useless. Does that mean that for one less output `getspendtx` succeeds but `setspendtx` fails?

Reply:
Yeah sorry, I forgot to paste it back after deleting the whole test. I'm adding the check that it's spent as well, so we check that there are no troubles *somewhere*